Assets/Scripts/Screeps3D/RoomObjects/Views/CreepView.cs

```diff
@@ -29,6 +30,19 @@ internal class CreepView : ObjectView
                 _horse.material.SetFloat("Magic", v);
         }
 
+        private void setDorito(bool isDorito) {
+            // _creepCore.enabled = !isDorito;
+            // _doritoCore.enabled = isDorito;
+            if(isDorito) {
+                _creepCore.transform.localScale = Vector3.zero;
+                _doritoCore.transform.localScale = Vector3.one;
+                _underLight.color = Color.red;
+            } else {
+                _doritoCore.transform.localScale = Vector3.zero;
+                _creepCore.transform.localScale = Vector3.one;
+            }
```

Comment:
Any particular reason for setting scale, instead of gameObject.SetActive(true/false) ?
Also, should we not restore the underlight color in the else part? i'm pretty sure creepviews are "cached" so a creep could potentially be spawned with red underlight 🤔

Reply:
just `enable/disable` on a root part (_doritoCore/_creepCor) was not propagating to child mesh renderers

Reply:
I see
enabled = true/false only enables / disables the script. e.g. in this case the Renderer. I've changed it to _creepCore.gameObject.SetActive(false/true) that way the entire tree is disabled :)